packages/avatars/stories/examples/MenuUsage.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story } from '@storybook/react';
+import {
+  Dropdown,
+  Trigger,
+  Menu,
+  MediaItem,
+  MediaFigure,
+  MediaBody,
+  ItemMeta
+} from '@zendeskgarden/react-dropdowns';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Button } from '@zendeskgarden/react-buttons';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+export const MenuUsage: Story = () => {
+  const [highlightedItem, setHighlightedItem] = useState<number | null>();
+
+  return (
+    <Grid>
+      <Row>
+        <Col>
+          <Dropdown onStateChange={changes => setHighlightedItem(changes.highlightedIndex)}>
+            <Trigger>
+              <Button>Default menu</Button>
+            </Trigger>
+            <Menu hasArrow>
+              <MediaItem value="linden">
+                <MediaFigure>
+                  <Avatar
+                    size="small"
```

Comment:
There is a rendering problem with this example. Why are the Avatars not being shown as small?
<img width="250" alt="Screen Shot 2020-09-16 at 9 48 34 AM" src="https://user-images.githubusercontent.com/143773/93367869-f5c1dc80-f801-11ea-887f-4b803848de82.png">

Reply:
These are rendering correctly for me in development and production builds

![image](https://user-images.githubusercontent.com/4030377/93371414-340dca80-f807-11ea-933d-d2f8fa525037.png)

Reply:
Hmm, deployment still off: https://5f62659c6de450374ab825a5--zendeskgarden-react-components.netlify.app/storybook/?path=/story/components-avatars--menu-usage

Reply:
I'm seeing the same rendering issues @jzempel is seeing with the Deployment link in an incognito browser window 🤔 .

Reply:
I found the issue! Both `react-menus` and `react-chrome` include styling rules for `img` elements. `Avatar` needs to increase it's specificity to beat out these competing styles.

This wasn't occurring in stylguidist as the order of style insertion was handled differently which avoided the specificity issues.